tslint.json

```diff
@@ -15,7 +15,7 @@
       "severity": "warn"
     },
     "eofline": true,
-    "forin": true,
+    "forin": false,
```

Comment:
why this change?

Reply:
Long, long ago in a JavaScript far away... libraries would modify the prototype of javascript objects in order to paper over the shoddy standard library. For example, it's entirely possible to set `Object.prototype.foo = () => console.log('foo');` and every object will then have a foo function. Potentially this means using a for in can be dangerous as you end up iterating keys you didn't expect. This linting rule checks that you have a `.hasOwnProperty` on the key. I've disabled it as it's not something that is done any more.

Reply:
okay cool